.travis.yml

```diff
@@ -7,9 +7,11 @@ env:
     # Use a global cache to re-use dependencies across release builds; downloading a mongodb release is quick.
     - CACHE_NAME=global_cache
   matrix:
-    - MONGODB_RELEASE=mongodb-linux-x86_64-ubuntu1204-3.0.14
-    - MONGODB_RELEASE=mongodb-linux-x86_64-ubuntu1204-3.2.12
-    - MONGODB_RELEASE=mongodb-linux-x86_64-ubuntu1204-3.4.2
+    - MONGODB_RELEASE=mongodb-linux-x86_64-ubuntu1404-3.0.15
```

Comment:
There are no binaries provided for 12.04 of newer versions of MongoDB, but from a quick inspection, Travis appears to be running 14.04 at this point, so I just updated all of the versions to download the release for that.